packages/grafana-ui/src/slate-plugins/suggestions.tsx

```diff
@@ -6,14 +6,7 @@ import { Editor as CoreEditor } from 'slate';
 import { Plugin as SlatePlugin } from '@grafana/slate-react';
 
 import TOKEN_MARK from './slate-prism/TOKEN_MARK';
-import {
-  makeFragment,
-  TypeaheadOutput,
-  CompletionItem,
-  TypeaheadInput,
-  SuggestionsState,
-  CompletionItemGroup,
-} from '..';
+import { makeFragment, TypeaheadOutput, CompletionItem, TypeaheadInput, SuggestionsState } from '..';
```

Comment:
Could you please update this import to be from the accurate path rather than `..`? We also need to introduce linter for such imports as they are problematic for Storybook.

Reply:
Sure